onyo/lib/onyo.py

```diff
@@ -357,28 +347,16 @@ def _get_root(self) -> Path:
         return root
 
     def _get_templates(self) -> set[Path]:
-        templates = {Path(file) for file in Path(self.root, ".onyo", "templates").glob('*') if
-                     Path(file).is_file() and not Path(file).name == ".anchor"}
-
-        return templates
+        from .utils import get_templates
+        return get_templates(self)
 
     #
     # HELPERS
     #
     @staticmethod
     def _is_protected_path(path: Union[Path, str]) -> bool:
-        """
```

Comment:
For `Repo._get_assets(self)` in line 280 you left the doc-string even though you did the same change with `from .utils import`

I personally think the docstrings should be in both functions until they get fully removed. They can either be full duplicates or just a shortened/summarized version, but having a function that does not have a doc-string in the repo-class might turn out to be inconvenient?

Reply:
True - docstring should stay. Will fix. 

Not having them copied in the extracted functions for now is intentional. I want to quickly iterate over refactoring them. So, adjusting docstrings all the time would be a major slowdown. And in this state, no direct usage of the extracted functions is intended. `Repo` is to be used as before for now, hence this is what still needs the docstrings.